infra/README.md

```
@@ -6,6 +6,7 @@
 ## 현재 구성된 리소스
 - IAM
     - uni-schedule-user: 관리자 권한(AdministratorAccess)을 가진 IAM User
+    - ecsTaskExecutionRole: ECS Task가 ECR에서 이미지 Pull 및 CloudWatch 로그를 기록할 수 있도록 하는 IAM Role
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)